play-java-rest-api-example/scripts/test-sbt

```diff
@@ -3,4 +3,7 @@
 echo "+----------------------------+"
 echo "| Executing tests using sbt  |" 
 echo "+----------------------------+"
-sbt ++2.12.8 test
+sbt ++$TRAVIS_SCALA_VERSION test
+
+# At least verifies that Gatling tests are compiling
+sbt ++$TRAVIS_SCALA_VERSION ";project gatling; gatling:compile"
```

Comment:
Oh, wow. That's a change...

Reply:
Hmmm, maybe I misundertstood.

Is this change required because of the change in the gatling version or was this a bug in the script (not running gatling and doing a noop `test`)?